script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector("form");
+const toDoInput = toDoForm.querySelector(".input-text");
+const toDoButton = toDoForm.querySelector(".input-button");
```

Comment:
```suggestion
const toDoInputText = toDoForm.querySelector(".input-text");
const toDoInputButton = toDoForm.querySelector(".input-button");
```
이러한 변수명도, 둘이 서로 구분가고 어떤 역할인지 명확한 것 같아요 !!

Reply:
오 ㅎㅎ! `querySelector`을 통해 가져오는 것이 DOM, 그니까 input-text의 클래스이름을 가진 HTML이어서 정확히 Text가 아니어서 Text는 안붙이는 게 나을 거 같아요!! 뒤에 `Button` 붙인 것은 좋은 생각 같아요! 감사합니다